cmd/template/cluster/provider/templates/aws/bastion_secret.yaml.tmpl

```diff
@@ -0,0 +1,64 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  labels:
+    cluster.x-k8s.io/cluster-name: {{ .Name }}
+    cluster.x-k8s.io/role: bastion
+    cluster.x-k8s.io/watch-filter: capi
+    giantswarm.io/cluster: {{ .Name }}
+    giantswarm.io/organization: {{ .Owner }}
+    "release.giantswarm.io/version": "{{ .ReleaseVersion }}"
+  name: {{ .Name }}-bastion
+  namespace: {{ .Namespace }}
+stringData:
+  value: |-
+    {
+       "ignition":{
+          "version":"2.2.0"
+       },
+       "passwd":{
+          "users":[
+             {
+                "name":"giantswarm",
+                "sshAuthorizedKeys":[
+                   "ssh-rsa AAAABEKf fake@giantswarm"
+                ],
+                "groups":[
+                   "sudo",
+                   "docker"
+                ],
+                "shell":"/bin/bash",
+                "uid":1000
+             }
+          ]
+       },
+       "storage":{
+          "files":[
+             {
+                "path":"/etc/hostname",
+                "filesystem":"root",
+                "mode": 420,
```

Comment:
Just to be sure, `420` seems like strange permissions to me. Is this a placeholder? Or is it just a thing that these templates use decimal notation that later gets turned into octal (which would be 644 which makes more sense).